spring-cloud-aws-sqs/src/test/java/io/awspring/cloud/sqs/integration/QueueAttributesResolverIntegrationTests.java

```diff
@@ -127,6 +127,31 @@ void shouldResolveFromUri() {
 			client.deleteQueue(DeleteQueueRequest.builder().queueUrl(queueUrl).build()).join();
 		}
 	}
+
+	@Test
```

Comment:
The test passes when running against local stack, but fails when I run against AWS:
`Caused by: software.amazon.awssdk.services.sqs.model.SqsException: The address https://sqs.us-east-1.amazonaws.com/ is not valid for this endpoint. (Service: Sqs, Status Code: 404, Request ID: 9d4c355f-c622-5eca-a4a3-6916d1f77ab5)`

If you can please set `useLocalStackClient` to off in the `BaseSqsIntegrationTest`class and re run it.

Reply:
@tomazfernandes  I have adapted the test so that it now also runs successfully against AWS. I am happy to receive feedback on whether the solution is acceptable.